server/graphql_schemas/event/schema.py

```diff
@@ -130,6 +132,7 @@ def create_event(category, user_profile, recurrence_event, **input):
     def mutate_and_get_payload(cls, root, info, **input):
         category_id = from_global_id(input.pop('category_id'))[1]
         recurrence_event = None
+        add_to_calendar = input.get('add_to_calendar')
```

Comment:
You can pass in True to be the default value if the add_to_calendar key is not found

Reply:
ah! how could I have missed that 🤦🏽‍♂️